src/main/java/mate/academy/service/CreateFileForReport.java

```diff
@@ -0,0 +1,5 @@
+package mate.academy.service;
+
+public interface CreateFileForReport {
```

Comment:
do we really need separate interface for creating a file? Can we create file while writing info in it?